modules/test/playwright/tests/headless-builder-web/get.spec.ts

```diff
@@ -170,6 +232,9 @@ test('can see path parameter property with map details', async ({
 
 	await headlessBuilderPage.goto();
 	await headlessBuilderPage.deleteApplication('My-app');
+	await objectAdminRestClient.objectDefinition.deleteObjectDefinition({
```

Comment:
Same here. Otherwise, if the test fails, we are leaving the object definitions in the database...